<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>888</width>
    <height>583</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="concentrationLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>1</number>
      </property>
      <property name="bottomMargin">
       <number>1</number>
      </property>
      <item>
       <widget class="QLabel" name="concentrationLabel">
        <property name="text">
         <string>Концентрация, Кмоль/м*3</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListView" name="concentrationListView"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="btnLayout">
      <item>
       <widget class="QPushButton" name="addBtn">
        <property name="text">
         <string>Добавить данные</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="removeBtn">
        <property name="text">
         <string>Удалить элемент(ы)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="removeAllBtn">
        <property name="text">
         <string>Удалить всё</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="plotBtn">
        <property name="text">
         <string>Построить графики</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="statusLabel">
        <property name="text">
         <string>Ожидание...</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="levelLayout" stretch="0,0">
      <property name="spacing">
       <number>5</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SizeConstraint::SetDefaultConstraint</enum>
      </property>
      <property name="leftMargin">
       <number>4</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="levelLabel">
        <property name="layoutDirection">
         <enum>Qt::LayoutDirection::LeftToRight</enum>
        </property>
        <property name="text">
         <string>Отклонение уровня, мм</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListView" name="levelListView">
        <property name="layoutDirection">
         <enum>Qt::LayoutDirection::LeftToRight</enum>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>888</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="fileMenu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="saveData"/>
    <addaction name="loadData"/>
   </widget>
   <widget class="QMenu" name="helpMenu">
    <property name="title">
     <string>Помощь</string>
    </property>
    <addaction name="information"/>
   </widget>
   <widget class="QMenu" name="serverMenu">
    <property name="title">
     <string>Сервер</string>
    </property>
    <addaction name="connectToServer"/>
   </widget>
   <addaction name="fileMenu"/>
   <addaction name="serverMenu"/>
   <addaction name="helpMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="saveData">
   <property name="text">
    <string>Сохранить данные</string>
   </property>
  </action>
  <action name="loadData">
   <property name="text">
    <string>Загрузить данные</string>
   </property>
  </action>
  <action name="connectToServer">
   <property name="text">
    <string>Подключиться к серверу</string>
   </property>
  </action>
  <action name="information">
   <property name="text">
    <string>Справка</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
